solr/contrib/ltr/src/test/org/apache/solr/ltr/TestLTRQParserPlugin.java

```diff
@@ -69,6 +69,20 @@ public void ltrModelIdDoesNotExistTest() throws Exception {
   }
 
   @Test
+  public void ltrBadRerankDocsTest() throws Exception {
+    final String solrQuery = "_query_:{!edismax qf='title' mm=100% v='bloomberg' tie=0.1}";
+    final SolrQuery query = new SolrQuery();
+    query.setQuery(solrQuery);
+    query.add("fl", "*, score");
+    query.add("rows", "4");
+    query.add("fv", "true");
+    query.add("rq", "{!ltr model=6029760550880411648 reRankDocs=-1}");
+
+    final String res = restTestHarness.query("/query" + query.toQueryString());
+    assert (res.contains("Must rerank at least 1 document"));
```

Comment:
minor: `assertTrue(res, res.contains("..."))` would include the full `res` in the test failure output if the tests fails, not sure if just a plain `assert` would do the same?